assets/src/edit-story/components/form/dropDown/list.js

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useState, useEffect, useCallback, useMemo, useRef } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { rgba } from 'polished';
+import { useDebouncedCallback } from 'use-debounce';
+
+/**
+ * Internal dependencies
+ */
+import { useKeyDownEffect } from '../../keyboard';
+
+const ListContainer = styled.div`
+  float: right;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  min-width: 160px;
+  max-height: 370px;
+  overflow-y: auto;
```

Comment:
Quick nit. I don't want to block this review on my timing. But if we do overflow-y here, it'd also be nice to set [overscroll-behavior](https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior).

Reply:
used `overscroll-behaviour: none auto`